pkg/test/resourcefixture/testdata/basic/edgecontainer/v1beta1/edgecontainercluster/create.yaml

```diff
@@ -0,0 +1,35 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: edgecontainer.cnrm.cloud.google.com/v1beta1
+kind: EdgecontainerCluster
+metadata:
+  name: edgecontainercluster-${uniqueId}
+  labels:
+    label-one: "value-one"
+spec:
+  projectRef:
+    external: ${projectId}
```

Comment:
Once we have the dependency project(see the comment below), we can reference to it by name here: `name: project-${uniqueId}`
Note that this will not reference to the project number, because you only specify "targetField: number" for `fleet.project`, not for `project`, in your service mapping.

Reply:
This part I am not sure. It seems for KCC to work it need to create SA account in the project and the workload identity binding on the KSA used by the KCC manager.
I think the test project {projectID} will have this setup in the test env, but I think that is not the case for the dependency project created dynamically.

Reply:
The question is, should `project` and `fleet.project` field using a same project? What happens if you specify ${projectID} for `project` and project-${uniqueId} for `fleet.project`?

Reply:
The project and fleet.project can be different. The first one is the project for hosting the cluster, the second one is the project that host the gke hub which can be different.

Reply:
Ah good! thanks for clarifying! Then I think it's fine to leave it as is. Thanks for making the code changes and addressing the comments!

Reply:
Thanks for reviewing the PR, and providing detailed instructions.